### DISCUSSION TITLE
配置-s严格模式后，音乐卡在加载

### DISCUSSION THREAD

**Discussion by Hutianming** [1715173329/UnblockNeteaseMusic#932] [Q&A] [answered]
配置严格模式-s后，音乐一直加载，服务器确实收到了请求信息，但一直卡在这里
![image](https://user-images.githubusercontent.com/50454065/202868194-511f1319-c2de-467f-9fda-1f48c9977019.png)
服务器收到请求
![image](https://user-images.githubusercontent.com/50454065/202868220-ddede61b-4668-4709-989c-adb6cff8f2db.png)
如果不配置-s严格模式。可以听是没问题的，但是服务器CPU和内存会炸，暂且没找到原因
node app.js -p 7777 -f 115.236.118.34 -o kuwo kugou -s  这个是开启应用的命令，不知道是不是需要进行额外配置

**Comment by 1715173329** [comment, marked as answer]
需要设置 endpoint，不然其他平台的流量过不去，详细参见其他 issue。

**Reply by Hutianming** [reply]
> 需要设置 endpoint，不然其他平台的流量过不去，详细参见其他 issue。

谢谢，加个-e参数解决问题，-e https://music.163.com
